releases/v1.5.0-rc.toml

```diff
@@ -89,5 +89,61 @@ containers the same set of permissions as the calling process, preventing errors
 when privileged containers attempt to get more permissions than given to the
 caller.
 
+#### Configuration changes
```

Comment:
this one https://github.com/containerd/containerd/pull/4399 should be mentioned I think :p